eng/tools/mcp/azure-sdk-java-mcp/src/brownfield-migrate.ts

```diff
@@ -3,17 +3,19 @@ import { CallToolResult } from "@modelcontextprotocol/sdk/types";
 export async function brownfieldMigration(): Promise<CallToolResult> {
     const cookbook = `Follow the instructions below to migrate the Java SDK to generate from TypeSpec.
 
+DO NOT SKIP STEP 3, IN ANY CONDITION.
+
 1. Initiate the Java SDK with the given URL to the "tspconfig.yaml" file.
    Use the tool to initiate the Java SDK, from tspconfig.yaml URL. This tool create a "tsp-location.yaml" file.
 
 2. Find the path to SDK module and its "pom.xml" file.
    Run "git status --porcelain" to find the "tsp-location.yaml" file. The path to the SDK module is the directory containing the "tsp-location.yaml" file. The "pom.xml" file is located in the same directory.
 
 3. Clean the Java source in SDK module.
-   Use the tool to clean the Java source in the SDK module.
+   Delete folder "src/main" in the SDK module. Delete folder "src/samples/**/generated" and "src/test/**/generated" in the SDK module.
 
-4. Synchronize the TypeSpec source for Java SDK.
-   Use the tool to synchronize the TypeSpec source for the Java SDK. This will download the the TypeSpec source specified in the "tsp-location.yaml" file.
+4. Generate the TypeSpec source for Java SDK.
```

Comment:
'Generate the TypeSpec source' -> 'Generate Java SDK'?

Reply:
updated